presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -85,6 +85,18 @@ Mathematical Functions
     The a, b parameters must be positive real numbers and value v must be a real value.
     The value v must lie on the interval [0, 1].
 
+.. function:: inverse_chi_squared_cdf(df, p) -> double
```

Comment:
Please put the function documentation in alphabetical order. The cdf documentation were in wrong places. You don't need to fix them in this PR, just make sure the new function documentations are added at the correct place.

Reply:
Given that we intend to add more distributions (t, F, Gamma, etc.). I think a better approach is to have a dedicated section in the doc called "Probability distributions" (similar to how the doc currently has "Statistical Functions", "Trigonometric Functions", etc.) 

I can do it in a separate diff after we finish adding all the distribution functions. Makes sense?

Reply:
Makes sense. You can add a separate PR to refactor the documentation first, and rebase this PR on top of that one. Or if you prefer to do that afterwards, let's put these functions in the conventional order for this PR and modify it later.